### ISSUE TITLE
Missing JavaScript Questions and Answers

### ISSUE THREAD

**Issue by anilectjose** [TharinduDilshan/InterviewQuestions2021#27]
I didn't see any JavaScript programs here. Can I add them?
Please give me a chance to help you.
@TharinduDilshan

**Comment by TharinduDilshan** [comment]
> I didn't see any JavaScript programs here. Can I add them? Please give me a chance to help you. @TharinduDilshan

Yes sure. Add a new folder called JavaScript and upload your coding question.
